deltabot_Processing/deltabot_Processing.pde

```diff
@@ -76,12 +76,12 @@ Slider servo3;
 
 void setup() {
   //open serial connection
-  myPort = new Serial(this, Serial.list()[0], 9600);
+  myPort = new Serial(this, Serial.list()[2], 9600);
```

Comment:
Changing this will break compatibility. Maybe move it up to a top-level constant and then it'll be easier for people to find and configure.